clinical-validation.wdl

```diff
@@ -46,142 +48,169 @@ workflow ClinicalValidation {
-            bedRegions = regions,
-            sample = sample,
-            dockerImage = dockerImages["rtg-tools"]
+        call gatk.SelectVariants as selectIndelsBaseline {
+            input:
+                referenceFasta = referenceFasta,
+                referenceFastaFai = referenceFastaFai,
+                referenceFastaDict = referenceFastaDict,
+                inputVcf = indexBaselineVcf.compressed,
+                inputVcfIndex = indexBaselineVcf.index,
+                selectTypeToInclude = "INDEL",
+                outputPath = unit.outputPrefix + "/baselineIndels.vcf.gz",
+                intervals = select_all([highConfidenceIntervals]),
+                dockerImage = dockerImages["gatk4"]
+        }
+
+        call rtg.Format as formatReference {
+            input:
+                inputFiles = [referenceFasta],
+                outputPath = unit.outputPrefix + "/reference.sdf",
+                dockerImage = dockerImages["rtg-tools"]
+        }
+
+        call rtg.VcfEval as evalSNPs {
+            input:
+                baseline = selectSNPsBaseline.outputVcf,
+                baselineIndex = selectSNPsBaseline.outputVcfIndex,
+                calls = selectSNPsCall.outputVcf,
+                callsIndex = selectSNPsCall.outputVcfIndex,
+                outputDir = unit.outputPrefix + "/evalSNPs/",
+                template = formatReference.sdf,
+                allRecords = true,
+                bedRegions = regions,
+                sample = unit.sampleNameVcf,
+                dockerImage = dockerImages["rtg-tools"]
+        }
+
+        call rtg.VcfEval as evalIndels {
+            input:
+                baseline = selectIndelsBaseline.outputVcf,
+                baselineIndex = selectIndelsBaseline.outputVcfIndex,
+                calls = selectIndelsCall.outputVcf,
+                callsIndex = selectIndelsCall.outputVcfIndex,
+                outputDir = unit.outputPrefix + "/evalIndels/",
+                template = formatReference.sdf,
+                allRecords = true,
+                bedRegions = regions,
+                sample = unit.sampleNameVcf,
+                dockerImage = dockerImages["rtg-tools"]
+        }
     }
 
     output {
-        Array[File] indelStats = evalIndels.allStats
-        Array[File] SNPStats = evalSNPs.allStats
-        File indelVcf = selectIndelsCall.outputVcf
-        File indelVcfIndex = selectIndelsCall.outputVcfIndex
-        File SNPVcf = selectSNPsCall.outputVcf
-        File SNPVcfIndex = selectSNPsCall.outputVcfIndex
-
-        File normalizedVcf = indexCallVcf.compressed
-        File normalizedVcfIndex = indexCallVcf.index
-
-        File BaselineIndelVcf = selectIndelsBaseline.outputVcf
-        File BaselineIndelVcfIndex = selectIndelsBaseline.outputVcfIndex
-        File BaselineSNPVcf = selectSNPsBaseline.outputVcf
-        File BaselineSNPVcfIndex = selectSNPsBaseline.outputVcfIndex
+        Array[Array[File]] indelStats = evalIndels.allStats
```

Comment:
A plain `Array[File]` will be easier to use. It is obtainable by using `flatten()`

Reply:
I've added flatten to the final output as suggested.